Kudu.Core/Scripts/kudusync

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-"$0.NET.exe" "$@"
+mono "$0.NET.exe" "$@"
```

Comment:
Gah I said the same thing but didn't commit my review :)

Windows or no, isn't this file already used _somewhere_ and working?